import ErrorProperty

let client = SeedErrorPropertyClient()

try await client.propertyBasedError.throwError(

)
